README.md

```diff
@@ -18,6 +18,15 @@ Rust appears to offer us the ability to implement fast, safer code with developm
 
 ## Goals
 
+Our priorities, higher priority items first, are:
+
+| What | Why |
+| --- | --- |
+| Safe, fast, memory-efficient | If it's not fast enough or it hogs memory nobody will ship it. We would accept unsafe - that we can convince ourselves to be safe - if it got us substantial perf. However, if we can have safety AND speed then it's a pretty easy choice to take that path. |
```

Comment:
Do we want to mention correctness here? The implication is our commitment to a solid test suite and the why matches the reasoning for time/space efficiency. 

If not, this lgtm as is.

Reply:
I aspired to call out aspects that are unique to this codebase. I worry that adding correctness or a test suite leads to documenting the entirety of software development best practices. To put it another way, I hope we take it as a given that we will do the basics like test suite, fuzzing, etc.